examples/tv-app/linux/include/content-launcher/ContentLauncherManager.h

```diff
@@ -51,11 +51,15 @@ class ContentLauncherManager : public ContentLauncherDelegate
     CHIP_ERROR HandleGetAcceptHeaderList(AttributeValueEncoder & aEncoder) override;
     uint32_t HandleGetSupportedStreamingProtocols() override;
 
+    uint32_t GetFeatureMap(chip::EndpointId endpoint) override;
+
 protected:
     std::list<std::string> mAcceptHeaderList;
     uint32_t mSupportedStreamingProtocols;
     std::vector<ContentEntry> mContentList;
 
 private:
     EndpointId mEndpointId;
+    // TODO: set this based upon meta data from app
+    uint32_t mDynamicEndpointFeatureMap = 3;
```

Comment:
So if this is modeled like a attribute read, I think we may have an opportunity to simplify this by sending this directly to the app as with other attributes. See PR [19924](https://github.com/project-chip/connectedhomeip/pull/19924) for this. In fact we could then do this for even the supported commands & attributes and wont have to get that info via manifest.
The other option is to get this info (as is in the manifest) when adding the content app and then caching it in the native layer. advantage of that is we dont have to send intent to the app which will then defer creating the app process till an actual command is received.

Reply:
Note that I am giving example from android platform but similar options may apply for linux as well.